### DISCUSSION TITLE
Issues with `DeepSource` and `Pageclip` Links on GitHub Education Pack

### DISCUSSION THREAD

**Discussion by Haleshot** [community/community#141144] [GitHub Education]
### Hi everyone,

I’ve encountered some issues with the [GitHub Education Pack](https://education.github.com/pack#github-copilot) when trying to access specific resources:

- The hyperlink for **DeepSource** (https://app.deepsource.com/gh-education) results in a **404 Not Found** page.
- The link for **Pageclip** (https://education.github.com/pack/redeem/pageclip-student) gives an **application error**. The error states:  
  > _"An error occurred in the application and your page could not be served. If you are the application owner, check your logs for details. You can do this from the Heroku CLI with the command heroku logs --tail."_  

Could these links be checked and updated? Thank you in advance for your help!

**Comment by balintdecsi** [comment]
Hi, is there any update? I'm running into the same problem. I haven't got answer from Pageclip support for a month now.
